File: matiasmasso/Mat.NET/DTO/DTOIntrastatException.cs
namespace DTO
{
    public class DTOIntrastatException
    {
        public Codis Codi { get; set; }
        public object Tag { get; set; }

        public enum Codis
        {
            NotSet,
            CodiMercancia,
            Weight,
            Amount
        }

        public DTOIntrastatException(Codis oCodi, object oTag)
        {
            Codi = oCodi;
            Tag = oTag;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOIntrastatException.vb
Public Class DTOIntrastatException
    Property Codi As Codis
    Property Tag As Object

    Enum Codis
        NotSet
        CodiMercancia
        Weight
        Amount
    End Enum

    Public Sub New(oCodi As Codis, oTag As Object)
        _Codi = oCodi
        _Tag = oTag
    End Sub
End Class